docs/advanced/advanced_troubleshooting.md

```diff
@@ -0,0 +1,124 @@
+# Troubleshooting
+Use this section for advanced troubleshooting information for your CSI driver.
+
+- [Verifying the CSI driver is running](#Verifying-the-CSI-driver-is-running)
+- [Multipath troubleshooting](#Multipath-troubleshooting)
+- [General troubleshooting](#General-troubleshooting)
+
+## Verifying the CSI driver is running
+
+  Verify that the CSI driver is running. (Make sure that all components are in the  _Running_ state), using the `$> kubectl get all -n <namespace> -l csi` command.
+
+## Multipath troubleshooting
+
+Use this information to help pinpoint potential causes for multipath failures.
+
+**Note:** Run these commands should be run on one of the worker nodes.
+
+-   **Display multipath information (FC and iSCSI)**
```

Comment:
what about NVME? should we even list the types of connectivity, even for procedures that are connectivity agnostic?

Reply:
@roysahar

Reply:
let's continue in https://github.com/IBM/ibm-block-csi-driver/pull/453
note that nvme is a part of FC configurations (unless we need separate info)